install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/changelog-focal

```diff
@@ -1,8 +1,8 @@
-securedrop-app-code (2.4.2~rc1+focal) focal; urgency=medium
+securedrop-app-code (2.4.2+focal) focal; urgency=medium
 
   * see changelog.md
 
- -- SecureDrop Team <securedrop@freedom.press>  Wed, 03 Aug 2022 14:26:30 -0400
+ -- Kunal Mehta <legoktm@debian.org>  Mon, 08 Aug 2022 21:54:36 -0400
```

Comment:
Should be updated to "SecureDrop Team <securedrop@freedom.press>" - historically we haven't used personal attributions in the changelog.

Reply:
Gah, not sure why I didn't notice this last night. Fixed!